stablehlo/dialect/StablehloOps.td

```diff
@@ -2371,7 +2371,8 @@ def StableHLO_GetDimensionSizeOp: StableHLO_Op<"get_dimension_size",
 }
 
 def StableHLO_MapOp: StableHLO_ShapedInterfaceOp<"map",
-      [RecursiveMemoryEffects, SameOperandsAndResultShape,
+      [RecursiveMemoryEffects, SameVariadicOperandSize /*map_c2*/,
```

Comment:
No need to add `SameVariadicOperandSize` if we do not have multiple variadic operands?

Reply:
`SameVariadicOperandSize` should be renamed to `VariadicOperandsHaveSameSize`. The singular `Operand` is misleading :)